monkestation/code/modules/jobs/job_types/explorer.dm

```diff
@@ -0,0 +1,75 @@
+/datum/job/explorer
+	title = JOB_LATEJOIN_EXPLORER
+	description = "Explore space. Salvage supplies. \
+		Visit strange places. Die in space."
+	department_head = list(JOB_HEAD_OF_PERSONNEL)
+	faction = FACTION_STATION
+	total_positions = 3
+	spawn_positions = 3
+	supervisors = SUPERVISOR_QM
+	exp_granted_type = EXP_TYPE_CREW
+	config_tag = "EXPLORER"
+
+	outfit = /datum/outfit/job/explorer
+	plasmaman_outfit = /datum/outfit/plasmaman/mining
+
+	paycheck = PAYCHECK_CREW
+	paycheck_department = ACCOUNT_CAR
+
+	mind_traits = list(TRAIT_DETECT_STORM)
+
+	display_order = JOB_LATEJOIN_EXPLORER
```

Comment:
this needs to be a number!!!

it's used for sorting, so this being a string is causing lots of runtimes:
![image](https://github.com/user-attachments/assets/a2691cb6-9ac6-458c-a970-7d67f3a2487a)

Reply:
fixed?